migrations/sql/V2019.02.21.9.59__add_address_columns_to_party.sql

```diff
@@ -0,0 +1,54 @@
+ALTER TABLE party ADD COLUMN suite_no       varchar(5);
+ALTER TABLE party ADD COLUMN address_line_1 varchar(50);
+ALTER TABLE party ADD COLUMN address_line_2 varchar(50);
+ALTER TABLE party ADD COLUMN city           varchar(50);
+ALTER TABLE party ADD COLUMN province_code  varchar(2);
+ALTER TABLE party ADD COLUMN postal_code    varchar(6);
+
+COMMENT ON TABLE party IS 'Party references an individual or an organization that does business with EMPR, and any relevant contact information, such as address.';
+
+CREATE TABLE IF NOT EXISTS province_code (
+    province_code    character varying(2)                   NOT NULL,
+    description      character varying(50)                  NOT NULL,
+    display_order    integer                                NOT NULL,
+    active_ind       boolean                  DEFAULT true  NOT NULL,
+    create_user      character varying(60)                  NOT NULL,
+    create_timestamp timestamp with time zone DEFAULT now() NOT NULL,
+    update_user      character varying(60)                  NOT NULL,
+    update_timestamp timestamp with time zone DEFAULT now() NOT NULL
+);
+
+ALTER TABLE province_code OWNER TO mds;
+COMMENT ON TABLE province_code IS 'Lookup table for valid province codes';
+
+
+INSERT INTO province_code
+    (
+    province_code,
+    description,
+    display_order,
+    create_user,
+    update_user
+    )
+VALUES
```

Comment:
do we want to set these values in alphabetical order?

Reply:
Sure. I just did it based on the SGC code in this table, since that's how it's ordered in this table:
https://www12.statcan.gc.ca/census-recensement/2011/ref/dict/table-tableau/table-tableau-8-eng.cfm

Given that the sort order is currently just used for dropdowns, I guess alphabetical probably makes the most sense. I'll update